src/app/clusters/groupcast-server/GroupcastServer.h

```diff
@@ -0,0 +1,55 @@
+/*
+ *    Copyright (c) 2025 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#pragma once
+
+#include "GroupcastLogic.h"
+#include <lib/core/DataModelTypes.h>
+#include <protocols/interaction_model/StatusCode.h>
+#include <app/server-cluster/DefaultServerCluster.h>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+
+/**
+ * @brief Provides coe-driven implementation for the Groupcast cluster server.
+          The `Instance()` method returns a reference to the global instance.
+ */
+class GroupcastCluster : public DefaultServerCluster
+{
+public:
+    static GroupcastCluster & Instance();
+
+    GroupcastCluster();
+
+    CHIP_ERROR Startup(ServerClusterContext & context) override;
+    void Shutdown() override;
```

Comment:
Don't override these if you do not need to (same for write attribute): DefaultServerCluster does the right thing and "noop" is not the right thing/override.